### PR TITLE
Changes `OnLeavePrompt` to `FeedbackPrompt`

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

**Comment by knod** [comment]
Hmm, I think this wasn't up to date with the latest dev. Hang on a sec.

**Comment by knod** [comment]
I am very confused.

**Comment by knod** [comment]
I apologize. I obviously thought I'd added changes that I hadn't.

File: src/containers/VisitPage.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ class VisitPage extends Component {
 
         <ConfirmLeave isBlocking={ this.state.isBlocking } />
         <FeedbackForm
-          isOpen={ this.state.feedbackOpen }
+          isOpen={ this.state.shouldShowFeedback }
```
Comment (ethanbb) [inline src/containers/VisitPage.js]: What's the reason for this change? "should show feedback" seems a little more vague to me than "feedback open" (the word "open" indicating that this is describing whether something is currently open or not, not just whether it "should" be).

Comment (knod) [reply src/containers/VisitPage.js]: So it's a fuzzy line, and here's my reasoning for what I did. I called it `shouldShowFeedback` because it isn't an indication that the feedback form is open. It happens before the feedback form opens and then the feedback form looks at it and decides for itself whether it wants to be open or not. Maybe it should be called `userWantsToGiveFeedback` or something that's less long that I can't figure out right now because I'm too tired. Does that make sense?

Comment (ethanbb) [reply src/containers/VisitPage.js]: Hm, maybe... It seems like in this case, the feedback form really is open if and only if `feedbackOpen` is true. But I guess you're saying that doesn't have to be the case? It sounds like you're trying to think of changing a state value as passing a message to components that care about it rather than mutating the whole app instantaneously, which is probably more accurate with what actually happens but also breaks the React abstraction in a way that I'm not sure is necessary here.

Comment (knod) [reply src/containers/VisitPage.js]: Just for the sake of posterity, the conclusion elsewhere was to call it `feedbackFormRequested` or something more like that.

**Comment by knod** [comment]
Apology not accepted! That was a great conversation! And I think it was one worth having.
